test_app/fullyIgnoredModule/src/androidTest/java/com/example/fullyIgnoredModule/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.fullyIgnoredModule
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
+    @Test
+    fun useAppContext() {
```

Comment:
The module name is fullyIgnoredModule but this test is not ignored

Reply:
nice catch